### PR TITLE
Milestone four

### PR DESCRIPTION
**Pull request by phalado**
In this Milestone, we implemented the filter for the books table.

### CODE REVIEW

File: src/reducers/filter.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const filter = (state = '', action) => {
+  switch (action.type) {
+    case 'CHANGE_FILTER':
+      return action.filter;
+    default:
+      return state;
+  }
+};
+
+export default filter;
```
Comment (shloch) [inline src/reducers/filter.js:10]: it should default to `ALL` [[requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/ItDefaultsToAll.png)]

File: src/containers/BookList.js [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
-const mapStateToProps = state => ({ books: state.books });
+const mapStateToProps = state => ({ books: state.books, filter: state.filter });
 
 const mapDispatchToProps = dispatch => ({
   deltBook: book => {
     dispatch(deleteBook(book));
   },
+  changeFilter: filter => {
+    dispatch(changeFilter(filter));
+  },
```
Comment (shloch) [inline src/containers/BookList.js:13]: great

**Review by shloch — APPROVED** [review]
## hello @phalado , @Grifo89 
Nice job...non point with the project execution

accomplished

File: src/reducers/filter.js (hunk 1 above)
Comment (phalado) [reply src/reducers/filter.js:10]: Thanks a lot. We will fix it.

**Pull request merged**